source/quickstart/osx.rst

```diff
@@ -6,31 +6,213 @@ Quickstart for Mac OSX
+.. admonition:: Network File System Volumes Break Consistency Guarantees
+   :class: note
+
+   MinIO's strict **read-after-write** and **list-after-write** consistency model requires local disk filesystems (``xfs``, ``ext4``, etc.).
+
+   MinIO cannot provide consistency guarantees if the underlying storage volumes are NFS or a similar network-attached storage volume. 
+
+   For deployments that *require* using network-attached storage, use NFSv4 for best results.
 
 Procedure
 ---------
 
-#. Step 1
+Install the MinIO Server
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Installation with `Homebrew <https://brew.sh>`_ is recommended.
+
+.. tab-set::
+
+   .. tab-item:: Homebrew
+
+      Open a Terminaal and run the following command to install the latest stable MinIO package using `Homebrew <https://brew.sh>`_.
+
+      .. code-block:: shell
+         :class: copyable
+
+         brew install minio/stable/minio
+
+      .. important::
+
+         If you previously installed the MinIO server using ``brew install minio``, then we recommend that you reinstall from ``minio/stable/minio`` instead.
+            
+         .. code-block:: shell
+            :class: copyable
+
+            brew uninstall minio
+            brew install minio/stable/minio
+            
+   .. tab-item:: Binary
+
+      Open a Terminal, then use the following commands to download the standalone MinIO server for MacOS and make it executable.
+            
+      .. code-block:: shell
+         :class: copyable
+
+         wget https://dl.min.io/server/minio/release/darwin-amd64/minio
+         chmod +x minio
+
+Test the :mc:`minio server`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Launch the :mc:`minio server` and use the :ref:`MinIO Console <minio-console>` to work with the MinIO volume.
+
+#. Launch the server
+   
+   From the Terminal, use this command to start a local MinIO instance in the ``/data`` folder.
+   Replace ``/data`` with the location to use for the MinIO instance.
+
+   .. code-block:: shell
+      :class: copyable
+
+      minio server /data
+
+   The :mc:`minio server` process prints its output to the system console, similar to the following:
+
+   .. code-block:: shell
+
+      API: http://192.0.2.10:9000  http://127.0.0.1:9000
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Console: http://192.0.2.10:9001 http://127.0.0.1:9001
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
+         $ mc alias set myminio http://192.0.2.10:9000 minioadmin minioadmin
+
+      Documentation: https://docs.min.io
+
+      WARNING: Detected default credentials 'minioadmin:minioadmin', we recommend that you change these values with 'MINIO_ROOT_USER' and 'MINIO_ROOT_PASSWORD' environment variables.
+
+#. Access the :ref:`minio-console` by going to a browser (such as Safari) and entering the Console address specified in the :mc:`minio server` command's output.
+   For example, http://127.0.0.1:9001.
```

Comment:
```suggestion
   For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console
```

Worth noting that the user _can_ direct their browser to the :9000 port. We will auto-redirect.

For a quickstart it might be sufficient to tell them to open a browser to 127.0.0.1:9000 since we know that will work.